calico/felix/refcount.py

```diff
@@ -97,6 +97,14 @@ def on_object_startup_complete(self, object_id, obj):
             _log.info("Ignoring on_object_startup_complete for instance "
                       "in state %s", obj.ref_mgmt_state)
             return
+            # MD4 It's not obvious to me why this branch is needed - I suspect it's because of asynchronous completion
+            # (thanks to actors) of startup while the manager thinks it's being (e.g.) destroyed, but this definitely
+            # needs commenting at least.
+
+            # MD4 On a related note, it wasn't clear to me what the intended way of
+            # covering these sort of things is.  The use of call_via_cb in the test code is
+            # very opaque - is there a reason you used these rather than something equivalent
+            # to the N-BASE's TB_SCHEDULER / SCHEDULE_ONE()?
```

Comment:
I think the ref counting stuff needed this more because the RefMgr actually starts other actors and I wanted to get some FV of that start process rather than fully dummy it out.  Elsewhere, I've used single-stepping.  You can create an actor and then run actor._step(), IIRC.